src/foundry/common/data/data.d.mts

```diff
@@ -213,62 +223,307 @@ declare namespace ShapeData {
   interface DefaultOptions {
     categories: ["IMAGE", "VIDEO"];
-    // initial: null;
+    // initial: {};
     wildcard: false;
     label: "";
   }
 
   interface Schema<SrcOptions extends FilePathFieldOptions> extends DataSchema {
     /**
      * The URL of the texture source.
+     * @defaultValue `initial.src ?? null`
      */
     src: fields.FilePathField<SrcOptions>;
 
     /**
-     * The scale of the texture in the X dimension.
+     * The X coordinate of the texture anchor.
+     * @defaultValue `initial.anchorX ?? 0`
      */
-    scaleX: fields.NumberField<{ nullable: false; initial: 1 }>;
+    anchorX: fields.NumberField<{ nullable: false; initial: number }>;
 
     /**
-     * The scale of the texture in the Y dimension.
+     * The Y coordinate of the texture anchor.
+     * @defaultValue `initial.anchorY ?? 0`
      */
-    scaleY: fields.NumberField<{ nullable: false; initial: 1 }>;
+    anchorY: fields.NumberField<{ nullable: false; initial: number }>;
 
     /**
      * The X offset of the texture with (0,0) in the top left.
+     * @defaultValue `initial.offsetX ?? 0`
      */
-    offsetX: fields.NumberField<{ nullable: false; integer: true; initial: 0 }>;
+    offsetX: fields.NumberField<{ nullable: false; integer: true; initial: number }>;
 
     /**
      * The Y offset of the texture with (0,0) in the top left.
+     * @defaultValue `initial.offsetY ?? 0`
+     */
+    offsetY: fields.NumberField<{ nullable: false; integer: true; initial: number }>;
+
+    /**
+     * @defaultValue `initial.fit ?? "fill"`
+     */
+    // TODO: report missing documentation
+    fit: fields.StringField<{ initial: string; choices: typeof CONST.TEXTURE_DATA_FIT_MODES }>;
+
+    /**
+     * The scale of the texture in the X dimension.
+     * @defaultValue `initial.scaleX ?? 1`
      */
-    offsetY: fields.NumberField<{ nullable: false; integer: true; initial: 0 }>;
+    scaleX: fields.NumberField<{ nullable: false; initial: number }>;
+
+    /**
+     * The scale of the texture in the Y dimension.
+     * @defaultValue `initial.scaleY ?? 1`
+     */
+    scaleY: fields.NumberField<{ nullable: false; initial: number }>;
 
     /**
      * An angle of rotation by which this texture is rotated around its center.
+     * @defaultValue `initial.rotation ?? 0`
      */
-    rotation: fields.AngleField;
+    rotation: fields.AngleField<{ initial: number }>;
+
+    /**
+     * The tint applied to the texture.
+     * @defaultValue `initial.tint ?? "#ffffff"`
+     */
+    tint: fields.ColorField<{ nullable: false; initial: string }>;
 
     /**
-     * An optional color string used to tint the texture.
+     * Only pixels with an alpha value at or above this value are consider solid
+     * w.r.t. to occlusion testing and light/weather blocking.
+     * @defaultValue `initial.alphaThreshold ?? 0`
      */
-    tint: fields.ColorField;
+    alphaThreshold: fields.AlphaField<{ nullable: false; initial: number }>;
+
+    // NOTE: continue here
```

Comment:
Is this a note from Foundry or to yourself?

Reply:
Woops, that was for myself and I forgot to remove it. I intend to do another pass and remove all of the TODOs anyway.